actions/clone-additional-plugins/action.yml

```diff
@@ -17,8 +17,12 @@ runs:
       if: inputs.ssh_private_key != ''
       shell: bash
       run: |
-        eval "$(ssh-agent)"
-        echo "${{ inputs.ssh_private_key }}" | ssh-add -
+        eval "$(ssh-agent)";
+        KEYS="${{ inputs.ssh_private_key }}";
+        export IFS="|";
```

Comment:
🧙🪄🔮

Reply:
Surprise surprise, it didn't work 😛. Now handled in Ruby